### PR TITLE
App 23 Tabbed view personal info

### PR DESCRIPTION
**Pull request by jacobjdavey**
I made five tabs with styles for the personal info page.

### CODE REVIEW

File: src/pages/PersonalInfo/PersonalInfo.module.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.tabs {
+  -webkit-tap-highlight-color: transparent;
```
Comment (jacoblurie29) [inline src/pages/PersonalInfo/PersonalInfo.module.css:2]: What is this? Just curious

File: src/pages/PersonalInfo/PersonalInfo.module.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  background: transparent;
+  margin: 10px 20px;
+  font-size: 90%;
+  padding: 6px 12px;
+  cursor: pointer;
+}
+
+.tabSelected {
+  border: none;
+  border-bottom: 2px solid #cf2e2e;
```
Comment (jacoblurie29) [inline src/pages/PersonalInfo/PersonalInfo.module.css:30]: Just a note about this, we will be making CSS variable that will replace the hardcoded colors soon

File: src/pages/PersonalInfo/PersonalInfo.tsx [hunk 3]
```diff
@@ -1,8 +1,33 @@
+import { useState, useEffect } from 'react';
+import styles from './PersonalInfo.module.css'
+import inter from './'
+
 const PersonalInfo = () => {
   // Personal info tab
   console.log("PersonalInfo");
```
Comment (jacoblurie29) [inline src/pages/PersonalInfo/PersonalInfo.tsx:7]: Delete this `console.log`

File: src/pages/PersonalInfo/PersonalInfo.tsx [hunk 4]
```diff
@@ -1,8 +1,33 @@
...
+  const [tabIndex, setTabIndex] = useState(0);
```
Comment (jacoblurie29) [inline src/pages/PersonalInfo/PersonalInfo.tsx:8]: Set the default as one
```suggestion
  const [tabIndex, setTabIndex] = useState(1);
```

File: src/pages/PersonalInfo/PersonalInfo.tsx [hunk 5]
```diff
@@ -1,8 +1,33 @@
...
+  const tabs = ["One", "Two", "Three", "Four", "Five"];
+
+  return (
+    <div className={styles.tabs}>
+      <link rel="stylesheet"
```
Comment (jacoblurie29) [inline src/pages/PersonalInfo/PersonalInfo.tsx:13]: Move this into the `<header>` sections of the `index.html` file in the root folder so the font applies to the whole project.

File: src/pages/PersonalInfo/PersonalInfo.module.css [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  padding: 6px 12px;
+  cursor: pointer;
+  font-weight: bold;
+}
+
+.tabPanel {
+  display: flex;
+  justify-content: center;
+  font-weight: bold;
+}
```
Comment (jacoblurie29) [inline src/pages/PersonalInfo/PersonalInfo.module.css:44]: Can we add some `:hover` behavior and maybe some animations?

**Easiest way to add animations:**
In the root component add the property: `transition: 200ms ease-in-out;`

_Note:  200ms is the time of animation and `ease-in-out` is just the type of movement behavior_

File: src/pages/PersonalInfo/PersonalInfo.tsx [hunk 7]
```diff
@@ -1,8 +1,33 @@
+import { useState, useEffect } from 'react';
+import styles from './PersonalInfo.module.css'
+import inter from './'
```
Comment (jacoblurie29) [inline src/pages/PersonalInfo/PersonalInfo.tsx:3]: Why this import?

File: src/pages/PersonalInfo/PersonalInfo.module.css [hunk 8]
```diff
@@ -0,0 +1,44 @@
+.tabs {
+  -webkit-tap-highlight-color: transparent;
+  width: 100%;
+  font-family: 'Inter', serif;
+}
+
+.tabList { /*applies to the entire box of tabs */
+  border-bottom: 1px solid gray;
+  margin: 0 0 10px; /*top right/left bottom-applies to separate the text for each tab from the tablist */
+  padding: 0;
```
Comment (jacoblurie29) [inline src/pages/PersonalInfo/PersonalInfo.module.css:10]: Not needed if it is just zero

**Review by jacoblurie29 — CHANGES_REQUESTED** [review]
This looks so sweet. See my comments so we can make it ready to merge.

File: src/pages/PersonalInfo/PersonalInfo.module.css (hunk 1 above)
Comment (jacobjdavey) [reply src/pages/PersonalInfo/PersonalInfo.module.css:2]: idk

File: src/pages/PersonalInfo/PersonalInfo.module.css (hunk 2 above)
Comment (jacobjdavey) [reply src/pages/PersonalInfo/PersonalInfo.module.css:30]: ok

**Review by jacoblurie29 — APPROVED** [review]
Great job!

**Pull request merged**